imap_processing/swe/l2/swe_l2.py

```diff
@@ -0,0 +1,31 @@
+"""SWE L2 processing module."""
+
+import numpy.typing as npt
+
+from imap_processing.swe.l1b.swe_l1b_science import read_lookup_table
+
+
+def calculate_particle_energy(esa_table_num: int) -> npt.NDArray:
+    """
+    Calculate particle energy.
+
+    To convert Volts to Energy, multiply ESA voltage in Volts by 4.75 to get
+    electron energy in eV.
+
+    Parameters
+    ----------
+    esa_table_num : int
+        ESA table number.
+
+    Returns
+    -------
+    energy : np.ndarray
+        720 step energy values.
+    """
+    # The lookup table gives voltage applied to analyzers.
+    esa_table = read_lookup_table(table_index_value=esa_table_num)
+
+    # Convert voltage to electron energy in eV by apply conversion factor.
+    conversion_factor = 4.75
```

Comment:
Is this a calibration factor? I have been recommending that folks put all calibration related constants in a single place for their instrument. Just something to think about.

Reply:
Good point. This would be perfect in instrument status summary file. I will add that as TODO in there.